src/components/ToggleSwitch/index.tsx

```diff
@@ -13,6 +13,7 @@ import {
 import dimen from '../../utils/dimen';
 import {normalizeFontSize} from '../../utils/fonts';
 import {colors} from '../../utils/colors';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` from the theme file is a good change. It helps to centralize color definitions and makes it easier to manage and update colors across the application.
```diff
+ import {COLORS} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->